hosts

```diff
@@ -0,0 +1,14 @@
+[ansible]
+control ansible_host=10.42.0.2
+
+[tower_node]
+tower ansible_host=10.42.0.3
```

Comment:
This is a bit different in that the other provisioners/workshops have students install Tower on the same box as Engine and does not (yet) support an option to install Tower automatically. This said, I believe it seems more logical for this node to be in the "control" group of the inventory.

Reply:
We shouldn't separate Tower into a new host.  Control should = both ansible CLI and Tower.

Reply:
Ansible and Tower host shouldn't be separated.  Same host, to be consistent with how this is taught.

Reply:
I agree that would be ideal they use different VM boxes. Can Vagrant handle that in one file or are you suggesting another Vagrantfile?

Reply:
Multi-box vagrant file is pretty straightforward.  Example: https://github.com/dfederlein/ansible-tower-demo/blob/master/Vagrantfile

Reply:
*Side note:* For automating Tower install on Vagrant box(es), localhost, remote, redundant, and with postgres master/replica see my previous work https://github.com/ing-bank/towering

Reply:
I added parameters for number of node (NODES) and their memory (NODEMEM). One host for ansible and Tower makes more sense indeed.